charmcraft/templates/init/config.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) {{ year }} {{ author }}
+# See LICENSE file for licensing details.
+options:
+  thing:
+    default: 🪄
```

Comment:
I definitely wouldn't default to a magic wand, it doesn't render on Github and many won't actually understand how it should be replaced.